### PR TITLE
documentation added for web3-eth-admin module & pendingTransactions

### PR DESCRIPTION
**Pull request by princesinha19**
## Description
I have added the documentation for **web3-eth-admin** module and **eth.pendingTranactions**.
It's dependent to #2655, #2657.

- [x] Documentation

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no warnings.
- [x] I have updated or added types for all modules I've changed
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran ```npm run test``` in the root folder with success and extended the tests if necessary.
- [x] I ran ```npm run build``` in the root folder and tested it in the browser and with node.
- [x] I ran ```npm run dtslint``` in the root folder and tested that all my types are correct
- [x] I have tested my code on an ethereum test network.

### CODE REVIEW

File: docs/web3-eth-admin.rst [hunk 1]
```diff
@@ -0,0 +1,423 @@
+.. _eth-admin:
+
+.. include:: include_announcement.rst
+
+========
+web3.eth.admin
+========
```
Comment (nivida) [inline docs/web3-eth-admin.rst]:
``` javascript 
==============
web3.eth.admin
==============
```

**Review by nivida — CHANGES_REQUESTED** [review]

File: docs/web3-eth-admin.rst (hunk 1 above)
Comment (princesinha19) [reply docs/web3-eth-admin.rst]: Done.

**Review by nivida — APPROVED** [review]

**Comment by nivida** [comment]
I will extend the docs a bit with some additional details. @princesinha19

**Comment by princesinha19** [comment]
okay, cool @nivida.

File: docs/web3-eth-admin.rst [hunk 2]
```diff
@@ -0,0 +1,423 @@
...
+
+1. ``Function`` - (optional) Optional callback, returns an error object as first parameter and the result as second.
+
+
+-------
+Returns
+-------
+
+
+``Promise<object>`` - The node information array.
```
Comment (joshstevens19) [inline docs/web3-eth-admin.rst:137]: Please detail the object call more with the object and a description next to each property.

File: docs/web3-eth-admin.rst [hunk 3]
```diff
@@ -0,0 +1,423 @@
...
+
+1. ``Function`` - (optional) Optional callback, returns an error object as first parameter and the result as second.
+
+
+-------
+Returns
+-------
+
+
+``Promise<ArrayBuffer>`` - List of all connected peers.
```
Comment (joshstevens19) [inline docs/web3-eth-admin.rst]: why is this a type of `ArrayBuffer` can it grow in size after it has been initialised?

File: docs/web3-eth-admin.rst [hunk 4]
```diff
@@ -0,0 +1,423 @@
...
+    web3.eth.admin.startRPC(host, port, cors, apis [, callback])
+
+It starts an HTTP based JSON RPC API webserver to handle client requests. All the parameters are optional.
+
+----------
+Parameters
+----------
+
+1. ``host`` - ``String`` - (optional) The network interface to open the listener socket on (defaults to "localhost").
+2. ``post`` - ``hex | number`` - (optional) The network port to open the listener socket on (defaults to 8545).
```
Comment (joshstevens19) [inline docs/web3-eth-admin.rst]: `string | number` as `hex` is misleading and people can see that as `string | number` so it reads `string | number | number`

Comment (princesinha19) [reply docs/web3-eth-admin.rst]: cool.

**Comment by princesinha19** [comment]
Done all the changes.

**Review by joshstevens19 — APPROVED** [review]

**Pull request merged**